src/components/UserMenu.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div>
```

Comment:
I'd love to see this as component in ODS .... let's chat about this later ...

Reply:
we can move this out later on, like the notifications drop down and others... but I still have mixed feelings about doing this due to the associated logic. let's consider separately